test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.cs

```diff
@@ -105,7 +106,7 @@ public void ConsoleAppModelBuilderCSFileContentRankingTest()
             Approvals.Verify(result.modelBuilderCSFileContent);
         }
 
-        [Fact]
+        [NotAppleSiliconFact("Model Builder not supported on Apple Silicon")    ]
```

Comment:
Is this still needed?

Reply:
Nope. I missed it when I removed the other ones. Gone now.